cardano_node_tests/tests/test_plutus_mint_build.py

```diff
@@ -47,6 +47,23 @@ def payment_addrs(
     return addrs
 
 
+def _check_plutus_cost(
+    plutus_cost: list, expected_tx_time: int, expected_tx_space: int, expected_lovelace: int
+):
+    """
+    Check plutus transaction cost.
+
+    units: the time is in picoseconds and the space is in bytes.
+    """
+    tx_time = plutus_cost[0]["executionUnits"]["steps"]
```

Comment:
The function should either:
1. accept data for single plutus script, instead of list of data and checking just first item, or
2. list of data for all plutus scripts (as it is now) and list of tuples of expected values